src/middlewared/middlewared/plugins/account.py

```diff
@@ -616,9 +616,9 @@ def do_create(self, data):
         verrors = ValidationErrors()
 
         if (
-            not data.get('group') and not data.get('group_create')
+            not data['group'] and not data['group_create']
         ) or (
-            data.get('group') is not None and data.get('group_create')
+            data['group'] is not None and data['group_create']
```

Comment:
Just wondering: why?  (And are you _**sure**_ these changes are safe?)

(And there's a bunch of these.)